### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by Pozharitskiy**
Finished all blocks and PP

### CODE REVIEW

**Comment by ElizavetaSelivanova** [comment]
`feature-1` - incorrect branch name

File: scss/components/_blog.scss [hunk 1]
```diff
@@ -28,8 +28,8 @@
   color: $black;
 }
 .blog-folders {
-  margin-left: 115px;
-  width: 100%;
+  width: 84%;
```
Comment (angelinabylitskaya) [inline scss/components/_blog.scss]: Do not use `width: 84%;`. Use only for 100%, 50%, etc

File: scss/components/_commercial.scss [hunk 2]
```diff
@@ -12,12 +12,12 @@
   height: 100%;
   padding-bottom: 215px;
 }
-.list-of-data-li {
+.list-of-data-list {
```
Comment (angelinabylitskaya) [inline scss/components/_commercial.scss]: Add empty line before each selector.

File: scss/components/_commercial.scss [hunk 3]
```diff
@@ -32,19 +32,16 @@
   @extend %list-letters;
 }
 
-.list-letters-two {
-  @extend %list-letters;
-  margin: 29px 58px 0 0;
+.two {
```
Comment (angelinabylitskaya) [inline scss/components/_commercial.scss]: Bad class name.

File: scss/components/_feedback.scss [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+  padding-top: 125px;
+  padding-bottom: 64px;
+}
+
+.feedback-headline {
+  font-size: 45px;
+  display: flex;
+}
+
+.feedback-underline {
```
Comment (angelinabylitskaya) [inline scss/components/_feedback.scss]: Consider using nesting here:
```
.feedback-underline {
  padding-bottom: 16px;
  border-bottom: 1px solid gray;
  display: flex;
  align-self: center;
  width: 100px;
  white-space: nowrap;
  margin-left: 137px;

  &-headline {
    margin: -18px 0 0 -3px;
    font-weight: bold;
  }

  &-text {
    margin: 105px 0 0px -101px;
    font-size: 18px;
  }
}
```
Do not use negative margin

File: scss/components/_blog.scss (hunk 1 above)
Comment (angelinabylitskaya) [reply scss/components/_blog.scss]: Here and in all other cases.

File: scss/components/_latest.scss [hunk 5]
```diff
@@ -0,0 +1,116 @@
+@import "headline";
```
Comment (angelinabylitskaya) [inline scss/components/_latest.scss]: Redundant imports.

File: scss/style.scss [hunk 6]
```diff
@@ -91,7 +93,7 @@ header li:hover {
 }
 
 .right-blog-arrow {
-  right: 59px;
+  right: -154px;
```
Comment (angelinabylitskaya) [inline scss/style.scss]: Only imports should be in style.scss.

File: scss/components/_latest.scss [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+@import "folder";
+@import "./mixins/flex-mix.scss";
+@import "./src/fonts";
+@import "./blog";
+
+.latest-portfolio {
+  background: $white;
```
Comment (angelinabylitskaya) [inline scss/components/_latest.scss]: Do not use color names for variable names.

File: scss/style.scss [hunk 8]
```diff
@@ -216,12 +218,69 @@ header li:hover {
...
+  list-style-type: none;
+  text-transform: uppercase;
+}
+
+.copyright {
+  margin-top: 67px;
+  width: 30%;
+  position: absolute;
+  right: 0;
+  color: #fafafa;
```
Comment (angelinabylitskaya) [inline scss/style.scss]: Move all colors to variables.

File: scss/components/_latest.scss (hunk 7 above)
Comment (Pozharitskiy) [reply scss/components/_latest.scss]: Done

File: scss/style.scss (hunk 8 above)
Comment (Pozharitskiy) [reply scss/style.scss]: Done

File: scss/components/_feedback.scss (hunk 4 above)
Comment (Pozharitskiy) [reply scss/components/_feedback.scss]: Fixed

File: scss/components/_commercial.scss (hunk 3 above)
Comment (Pozharitskiy) [reply scss/components/_commercial.scss]: Got it

File: scss/components/_blog.scss (hunk 1 above)
Comment (Pozharitskiy) [reply scss/components/_blog.scss]: Got it

**Pull request merged**